File: array/1550_three_consecutive_odds/1550_three_consecutive_odds.go
package main

import "fmt"

func threeConsecutiveOdds(arr []int) bool {
	cnt := 0
	for _, n := range arr {
		if n&1 == 1 {
			cnt++
		} else {
			cnt = 0
		}
		if cnt == 3 {
			return true
		}
	}
	return false

}

func main() {
	arr := []int{1, 2, 34, 3, 4, 5, 7, 23, 12}
	res := threeConsecutiveOdds(arr)
	fmt.Println(res)
}

File: array/1550_three_consecutive_odds/1550_three_consecutive_odds.py
from typing import List

class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        cnt = 0
        for i in arr:
            if i & 1:
                cnt += 1
            else:
                cnt = 0
            if cnt == 3:
                return True
        return False


arr = [2,6,4,1]
res = Solution().threeConsecutiveOdds(arr)
print(res)